src/components/Navbar.tsx

```diff
@@ -40,10 +40,10 @@ const Navbar = () => {
       variant="dark"
       navItems={[
         {
-          children: getDropdownListOfPages(hambergerPages),
+          children: getDropdownListOfPages(pages),
           label: '',
           type: 'link-list',
-          className: 'nav-hamberger pr-4 d-md-none',
+          className: 'pr-4 d-md-none',
```

Comment:
Could you take a look at the failed tests in src/__tests__/components/Navbar.test.tsx?

Reply:
Yes, I will have a look and report as soon as resolved :)